index.html

```diff
@@ -50,32 +50,32 @@ <h1>
         <div class="social-media-headline">
           <ul class="links">
             <li class="link-item">
-              <a rel="noopener" target="_blank" href="https://github.com/Javivy"><img src="icons/Vector.png" alt="Github" /></a>
+              <a href="#"><img src="icons/Vector.png" alt="Github" /></a>
             </li>
             <li class="link-item">
-              <a rel="noopener" target="_blank" href="https://www.linkedin.com/in/javier-alejandro-vargas-ortega-5998a5212/"><img src="icons/Frame.png" alt="Linkedin" /></a>
+              <a href="#"><img src="icons/Frame.png" alt="Linkedin" /></a>
             </li>
             <li class="link-item">
-              <a rel="noopener" target="_blank" href="#"><img src="icons/Vector-hand.png" alt="Icon" /></a>
+              <a href="#"><img src="icons/Vector-hand.png" alt="Icon" /></a>
             </li>
             <li class="link-item">
-              <a rel="noopener" target="_blank" href="https://twitter.com/Javivy_2803"><img src="icons/twitter.png" alt="Twitter" /></a>
+              <a href="#"><img src="icons/twitter.png" alt="Twitter" /></a>
             </li>
             <li class="link-item">
-              <a rel="noopener" target="_blank" href="#"><img src="icons/m-icon.png" alt="Icon" /></a>
+              <a href="#"><img src="icons/m-icon.png" alt="Icon" /></a>
             </li>
           </ul>
         </div>
         <div class="chevron-down">
-          <a href="#recent-work"><img src="icons/chevron-down.png" alt="Icon" /></a>
+          <a href="#"><img src="icons/chevron-down.png" alt="Icon" /></a>
         </div>
       </div>
     </header>
 
     <main>
       <!-- RECENT WORKS SECTION -->
 
-      <section id="recent-work" class="works-section">
+      <section class="works-section">
```

Comment:
- [ ] Kindly ensure that your webpage has a minimum contrast ratio of at least 4.5:1.  You can use [this aditus tool](https://www.aditus.io/button-contrast-checker/) to help to check the color contrast of the project 👍🏽 .

Here is a failing result of the contrast check. Kindly fix it 👍🏽 

![image](https://user-images.githubusercontent.com/63473584/187311250-00748ff6-d6e2-4018-b054-09552dc0ce17.png)

![image](https://user-images.githubusercontent.com/63473584/187311308-59b3f486-d1dc-4937-9cfd-0b99d0568fdb.png)

Reply:
@vickymarz Hi, shouldn't I leave it like that since the design uses those colors?

Reply:
Hello @vickymarz thank you for reviewing our work! We had that discussion during our review, should we change the colors from the design to match the accessibility goals or should we stick to the design template from Figma? Please let us know to work towards the purpose of the project!

Reply:
@OscarFM-dev as regards the button, your hover color and your default color looks the same. you should implement the hover state to at least improve accessibility